### ISSUE TITLE
elastic search

### ISSUE THREAD

**Issue by atherdon** [ChickenKyiv/recipe-search-react#39]

**Comment by atherdon** [comment]
this looks like task, that you'll adore too

**Comment by chauhannishith** [comment]
Can you tell me more about what i have to do in this, so i can find precisely

**Comment by atherdon** [comment]
of course.
So we're building a search form. that will be query our server. and react + server will works fine for this moment. Because we don't have a lot of data in our databases. I mean we don't have a lot of recipes. So search will works normal. but if we'll find a way to add a lot of recipes into our database - search will be a weak part of our logic. because we don't have indexes, etc. So i think that using Elasticsearch will be a good approach. at least we should try and figure it out.

So if we have 50-500 recipes - and user will query "chicken" - our search will work fine.
but if we'll have 2-10k or rows - it'll be a huge problem and require a lot of power of our servers.

**Comment by chauhannishith** [comment]
Do you have any sample for this? Like how to connect to db and send the query

**Comment by atherdon** [comment]
no, i don't have a proper example. and i'm not sure that this is a good
task to work on.
first of all i'm started to think that is loopback not are the best choice
from the market. and it's good, because i didn't know a lot about this 1
year ago. i was thinking about moving to graphQL soon or later, because it
also have an ORM plugins and looks like more fast evolving thing.
i also exploring the way to move recipe storing into blockain technology,
because this is 'hot' topic right now and  maybe this will give our users
ability to make copy-right claims - because right now a lot of recipes are
stolen from their owners.
